packages/ui/react-ui-geo/src/components/Map/Map.tsx

```diff
@@ -117,7 +145,15 @@ const MapCanvas = forwardRef<MapController, MapCanvasProps>(({ markers, center,
   }, [markers]);
 
   return (
-    <div ref={ref} className='flex w-full h-full overflow-hidden bg-baseSurface'>
+    <div ref={ref} role='none' className='grid inset-0 overflow-hidden bg-baseSurface'>
+      {/* Focus ring. */}
+      <div
+        className={mx(
+          'z-[9999] absolute inset-0 border border-transparent pointer-events-none',
```

Comment:
This is a wild z-index to set. Is it necessary? It may cause bugs.

Highly recommend just using one of the `dx-focus-ring` classes rather than reinventing this wheel.

Reply:
@thure given the TZ difference, please suggest how to do that -- i struggled to get it working and it's not documented or used in a story that enabled me to figure out how to use it.